webapp/components/post_view/components/post_list.jsx

```diff
@@ -139,6 +132,15 @@ export default class PostList extends React.Component {
             });
         }
 
+        // Postpone all DOM related calculations to next frame.
+        // scrollHeight etc might return wrong data at this point
```

Comment:
It means that _before the timeout_ the dimensions of some DOM elements can be wrong, thats why there is a timeout to ensure we use the correct dimensions when determining if the user has hit the bottom of the list.

while developing I focused on mobile and several different attachment types. You might want to check extra for desktop browsers? But I assume this still works the same way. :)